examples/notebooks/healthcare/beam_nlp.ipynb

```diff
@@ -0,0 +1,968 @@
+        "from apache_beam.options.pipeline_options import PipelineOptions\n",
+        "if debug:\n",
+        "    runnertype = \"InteractiveRunner\"\n",
+        "else:\n",
+        "    runnertype = \"DataflowRunner\"\n",
+        "\n",
+        "options = PipelineOptions(\n",
+        "    flags=argv,\n",
+        "    runner=runnertype,\n",
+        "    project=PROJECT,\n",
+        "    job_name=\"my-healthcare-nlp-job\",\n",
+        "    temp_location=TEMP_LOCATION,\n",
+        "    region=LOCATION)"
+      ],
+      "metadata": {
+        "id": "07ct6kf55ihP"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following defines a `PTransform` named `ReadLinesFromText`, that extracts lines from a file."
+      ],
+      "metadata": {
+        "id": "dO1A9_WK5lb4"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "class ReadLinesFromText(beam.PTransform):\n",
+        "\n",
+        "    def __init__(self, file_pattern):\n",
+        "        self._file_pattern = file_pattern\n",
+        "\n",
+        "    def expand(self, pcoll):\n",
+        "        return (pcoll.pipeline\n",
+        "                | beam.io.ReadFromText(self._file_pattern))"
+      ],
+      "metadata": {
+        "id": "t5iDRKMK5n_B"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following sets up an Apache Beam pipeline with the *Interactive Runner*. The *Interactive Runner* is the runner suitable for running in notebooks. A runner is an execution engine for Apache Beam pipelines."
+      ],
+      "metadata": {
+        "id": "HI_HVB185sMQ"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from google.auth import default\n",
+        "\n",
+        "credentials = default()"
+      ],
+      "metadata": {
+        "id": "dMc10Dlgtp1c"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "p = beam.Pipeline(options = options)"
+      ],
+      "metadata": {
+        "id": "7osCZ1om5ql0"
+      },
+      "execution_count": null,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "The following sets up a PTransform that extracts words from a Google Cloud Storage file that contains lines with each line containing a In our example, each line is a medical notes excerpt that will be passed through the Healthcare NLP API\n",
+        "\n",
+        "| is an overloaded operator that applies a PTransform to a PCollection to produce a new PCollection. Together with |, >> allows you to optionally name a PTransform.\n",
+        "\n",
+        "Usage: [PCollection] | [PTransform] or [PCollection] | [name] >> [PTransform]"
```

Comment:
Could we separate these examples onto 2 different lines (and maybe bold "or")? This was confusing on my first readthrough because I thought you were saying

`[PCollection] | (either [PTransform] or [PCollection]) | [name] >> [PTransform]"`

and that didn't make sense to me